lib/Github/Api/AbstractApi.php

```diff
@@ -20,6 +20,13 @@
     protected $client;
 
     /**
+     * The requested page (GitHub pagination).
+     *
+     * @var null|int
+     */
+    protected $page;
```

Comment:
Why not private?

Reply:
ping @Gummibeer, please change this property to private